resources/views/filament/components/server-console.blade.php

```diff
@@ -114,6 +114,19 @@ class="w-full focus:outline-none focus:ring-0 border-none"
         const socket = new WebSocket("{{ $this->getSocket() }}");
         let token = '{{ $this->getToken() }}';
 
+        socket.onerror = function(websocketErrorEvent) {
+            @php
+                $alerts = session()->get('alert-banners', []);
+                $alerts[] = [
+                    'title' => 'Could not connect to websocket!',
+                    'body' => 'Check your browser console for more details.',
```

Comment:
Can't we get the error message from `websocketErrorEvent`?

Reply:
Sadly no, it's just a generic event without additional data.